src/pds/ingress/authorizer/index.js

```diff
@@ -20,8 +22,8 @@
  * https://github.com/awslabs/aws-jwt-verify
  *
  */
-exports.handler =  async(event, _context, callback) => {
-    let token = event.authorizationToken;
+exports.handler = async(event, _context, callback) => {
+    let token = event.headers.Authorization;
     console.log(token);
```

Comment:
Better to remove this log message, because even though access tokens are short lived, they should be considered as confidential.